apps/sherbet_service/lib/sherbet.service/contact/communication/method/mobile.ex

```diff
@@ -208,8 +209,8 @@ defmodule Sherbet.Service.Contact.Communication.Method.Mobile do
         end
     end
 
-    defp generate_key() do
-        0..5
+    defp generate_key(data \\ @key) do
+        0..data
```

Comment:
Same as email, `data` could be renamed to `length`. Aside from that, it looks good.